caffe2/core/module.cc

```diff
@@ -0,0 +1,122 @@
+#include "caffe2/core/logging.h"
```

Comment:
Should this have Facebook license boiler plate here?

Reply:
Ah yeah, should be.